src/frontend/src/components/Wallet.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <div class="container">
+    <h1>{{ walletAmount }} BSC</h1>
+  </div>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from "vue-property-decorator";
+
+@Component
+export default class Wallet extends Vue {
+  get walletAmount() {
+    return this.$store.state.walletAmount;
+  }
+
+  walletId() {
+    return this.$store.state.walletId;
+  }
+
+  privateKey() {
+    return this.$store.state.privateKey;
+  }
+
+  walletCreated() {
+    return this.$store.state.walletCreated;
+  }
```

Comment:
Are we ever going to need these methods for wallet component? If not we can probably remove them!

Reply:
We'll need walletCreated for the time being until we initialize wallet on account creation and not in Wallet component. I'll change it to use a getter tho